metpy/calc/thermo.py

```diff
@@ -1038,3 +1038,63 @@ def most_unstable_parcel(p, temperature, dewpoint, heights=None,
     theta_e = equivalent_potential_temperature(p_layer, T_layer)
     max_idx = np.argmax(theta_e)
     return p_layer[max_idx], T_layer[max_idx], Td_layer[max_idx]
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def sbcape_sbcin(pressure, temperature, dewpt, profile=None):
```

Comment:
The idea with the profile argument was to allow the use to pass in a profile (say if it had been calculated for plotting already) to reduce the number of required function calls.